src/server/getters/get-reporting-fees.ts

```diff
@@ -62,6 +101,127 @@ interface RepStakeResults {
   lostRep: BigNumber;
 }
 
+function getUniverseAndParentUniverse(db: Knex, reporter: Address, universe: Address|null, feeWindow: Address|null, callback: (err: Error|null, result?: UniverseAndParentUniverse) => void) {
+  const query = db("fee_windows")
+  .select("universes.universe", "universes.parentUniverse")
+  .join("universes", "fee_windows.universe", "universes.universe")
+  .groupBy("fee_windows.universe");
+  if (universe != null) {
+    query.where("fee_windows.universe", universe);
+  } else if (feeWindow != null) {
+    query.where("fee_windows.feeWindow", feeWindow);
+  }
+  query.asCallback((err, universeAndParentUniverse: Array<UniverseAndParentUniverse>) => {
+    if (err) return callback(err);
+    return callback(null, universeAndParentUniverse[0]);
+  });
+}
+
+function formatMarketInfo(marketParticipants: any, parentUniverse: Address, callback: (err: Error|null, formattedMarketInfo?: FormattedMarketInfo) => void) {
+  let forkedMarket = {} as ForkedMarket;
```

Comment:
This isn't an async call, can we convert it to a non-callback call?

Reply:
Yeah, this shouldn't be an async call. I'll change it.